### PR TITLE
Verify C# code in the CI

### PR DESCRIPTION
**Pull request by swift-kim**
Add the dotnet-format job and dotnet-test job to the CI.

### CODE REVIEW

File: .github/workflows/analyze.yml [hunk 1]
```diff
@@ -21,18 +21,12 @@ jobs:
...
-        run: |
-          git clone --depth=1 https://github.com/flutter/flutter.git
-          cd flutter
-          git fetch --depth=1 https://github.com/flutter/flutter.git $(cat ../bin/internal/flutter.version)
-          git checkout FETCH_HEAD
-      - name: Run tests
-        run: flutter/bin/flutter test
+      - uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '6.0.x'
```
Comment (swift-kim) [inline .github/workflows/analyze.yml:30]: [dotnet-format](https://github.com/dotnet/format) is part of the .NET 6 SDK. Another option is to use .NET 5 and install the `dotnet-format` global tool.

**Comment by WonyoungChoi** [comment]
How about changing a setting of `.editorconfig` as follows?
```js
dotnet_sort_system_directives_first = true
```

**Comment by swift-kim** [comment]
@WonyoungChoi Done!

**Review by WonyoungChoi — APPROVED** [review]
LGTM

**Pull request merged**